jobs/travis.rb

```diff
@@ -74,40 +72,15 @@
 						'title'=>branch['finished_at'],
 						'result'=>branch['state'],
 						'url'=> 'https://travis-ci.org/%s/builds/%d' % [repo_slug,branch['id']]
-					} 
-				end
-			
+					}
+        end
```

Comment:
`end` at 76, 8 is not aligned with `items = repo_branches['branches']` at 51, 3 or `.map do |branch|` at 66, 4